woocommerce-gateway-stripe.php

```diff
@@ -403,10 +401,8 @@ public function filter_gateway_order_admin( $sections ) {
 				unset( $sections['stripe_sepa'] );
 				unset( $sections['stripe_multibanco'] );
 
-				$sections['stripe'] = 'Stripe';
-				if ( WC_Stripe_Feature_Flags::is_upe_preview_enabled() ) {
-					$sections['stripe_upe'] = 'Stripe checkout experience';
-				}
+				$sections['stripe']            = 'Stripe';
+				$sections['stripe_upe']        = 'Stripe checkout experience';
```

Comment:
I know they already weren't translated, but maybe we should go ahead and pass these through the i18n function?